tests/storage/test_id_generators.py

```diff
@@ -259,8 +259,9 @@ def test_empty(self) -> None:
 
         id_gen = self._create_id_generator()
 
-        # The table is empty so we expect an empty map for positions
-        self.assertEqual(id_gen.get_positions(), {})
+        # The table is empty so we expect the map for positions to have a dummy
+        # minimum value.
+        self.assertEqual(id_gen.get_positions(), {"master": 1})
```

Comment:
Is 1 the right sentinel here? (At the bottom of the diff we use -1 for something that looks similar)

Reply:
Yeah, so `1` is what is returned by `get_current_token()` for the empty case. The reason for `-1` below is that that is testing the *negative* stream IDs (i.e. backfill which produces negative stream IDs)